Gems/PhysX/Code/Source/StaticRigidBodyComponent.h

```diff
@@ -40,26 +44,79 @@ namespace PhysX
         static void GetIncompatibleServices(AZ::ComponentDescriptor::DependencyArrayType& incompatible);
         static void GetDependentServices(AZ::ComponentDescriptor::DependencyArrayType& dependent);
 
-        // AzPhysics::SimulatedBodyComponentRequestsBus::Handler overrides ...
+        // RigidBodyRequests + SimulatedBodyComponentRequests overrides ...
         void EnablePhysics() override;
         void DisablePhysics() override;
         bool IsPhysicsEnabled() const override;
+
+        AzPhysics::SceneQueryHit RayCast(const AzPhysics::RayCastRequest& request) override;
         AZ::Aabb GetAabb() const override;
+
+        // AzPhysics::SimulatedBodyComponentRequests overrides ...
         AzPhysics::SimulatedBodyHandle GetSimulatedBodyHandle() const override;
         AzPhysics::SimulatedBody* GetSimulatedBody() override;
```

Comment:
suggestion: Same as before, might be worth just grouping these functions and not having two comment blocks

Reply:
Answered in the previous comment.